src/m365/entra/commands/m365group/m365group-set.spec.ts

```diff
@@ -50,6 +58,7 @@ describe(commands.M365GROUP_SET, () => {
     loggerLogSpy = sinon.spy(logger, 'log');
     loggerLogToStderrSpy = sinon.spy(logger, 'logToStderr');
     (command as any).pollingInterval = 0;
+    sinon.stub(accessToken, 'isAppOnlyAccessToken').returns(true);
```

Comment:
I think we should use delegated permissions by default, and create some tests for application permissions since they have more restrictions.